controllers/MessageController.ts

```diff
@@ -41,13 +41,19 @@ export default class MessageController implements MessageControllerI {
                 .findAllMessagesReceivedByUser);
```

Comment:
This comment is for the whole getInstance method - I think we need to go over the URL patterns. There are some inconsistencies that make the API kind of confusing and I don't think it follows the rest pattern in some places. Let's chat sometime tomorrow and decide how to update them?